drivers/bluetooth/custom/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 add_subdirectory_ifdef(CONFIG_BT_RS9116W    rs9116w)
+
+private(pet)
```

Comment:
Remember to remove this before pushing. We might want to have some way to test this publicly by adding another magic word that can be public